ert/data/record/_record.py

```diff
@@ -466,38 +471,31 @@ def collection_type(self) -> RecordCollectionType:
 
 
 async def load_collection_from_file(
-    file_path: pathlib.Path,
-    mime: str,
+    transformation: "RecordTransformation",
     length: int = 1,
-    is_directory: bool = False,
-    smry_keys: Optional[List[str]] = None,
+    root_path: pathlib.Path = pathlib.Path(),
```

Comment:
Make optional, needs to be None as an object in initializer list is an anti-pattern.

Reply:
It's immutable, so we've concluded that it's fine as it is.